src/hotspot/share/gc/shared/cardTable.cpp

```diff
@@ -428,7 +454,7 @@ MemRegion CardTable::dirty_card_range_after_reset(MemRegion mr,
 }
 
 uintx CardTable::ct_max_alignment_constraint() {
-  return card_size * os::vm_page_size();
+  return GCCardSizeInBytes * os::vm_page_size();
```

Comment:
The reason why `card_size` can't be used here is that `card_size` haven't been set when this method is called, correct? If so, please put a comment here.